ocaml/xapi/repository.ml

```diff
@@ -716,70 +702,20 @@ let set_pending_restart_device_models ~__context ~host =
       (* No device models are running for this VM *)
       None
 
-let restart_device_models ~__context ~host =
-  (* Restart device models of all running HVM VMs on the host by doing
-   * local migrations. *)
-  do_with_device_models ~__context ~host ~action_label:"restart"
-  @@ fun (ref, record) ->
+let set_recommended_restart_device_models ~__context ~host =
```

Comment:
This should be able to be combined with `set_pending_restart_device_models` as most bits of them are same.

Reply:
combined